src/blockchain/shelley/certificate.js

```diff
@@ -23,19 +27,10 @@ export type PoolUpdate = {
   pool_id: PoolId,
 }
 
-export type StakeDelegation = {
-  type: 'StakeDelegation',
-  // null here signifies NonDelegated, ie un-delegate
-  pool_id: ?PoolId,
-  account: string,
-}
-
-export type OwnerStakeDelegation = {
-  type: 'OwnerStakeDelegation',
-  // null here signifies NonDelegated, ie un-delegate
-  pool_id: ?PoolId,
-  // this is not in the certificate, but was inferred from the tx
+export type StakeDelegationType = {
```

Comment:
Why was this changed? If we're going to be combining `OwnerStakeDelegation` and `StakeDelegation` we can't use the `type` field as we're already using this as the tag in the `CertificateType` tagged enum.